### ISSUE TITLE
Controller version 5.10.20, not 5.10.21...?

### ISSUE THREAD

**Issue by dolphinkite** [jacobalberty/unifi-docker#194]
### Host operating system

Synology NAS

### What tag are you using

Latest

### What complete docker command or docker-compose.yml do you use to launch the container (omitting sensitive values)?

Hi,

I am running the latest Docker container that was updated 11 days ago. The login screen shows that the controller version is 5.10.20, but the description [here](https://hub.docker.com/r/jacobalberty/unifi)  indicate that the bundled version should be 5.10.21.

When I login the controller, I get a message on the top right corner telling me that a new version (5.10.21) of the controller is available, and when I click to download it, I download a .deb file (Debian..?) which I don't think is what I want.

So, I'm confused. Is this container running version 5.10.20 or 5.10.21? How can I upgrade to 5.10.21?

Thanks

**Comment by jacobalberty** [comment]
You may need to do a fresh pull of the image. I just checked and the login screen for jacobalberty/unifi:latest  is definitely showing 5.10.21 for me.

**Comment by dolphinkite** [comment]
Humm. I had **definitely** tried to do a `docker pull jacobalberty/unifi:latest` and delete my old container and run a new one, but the version still showed 5.10.20.

I just manually deleted each image (not container) that was on my system, did a new pull. Indeed I see version 21 now.

One thing that I noticed is that I recently switched from "latest" to "stable" thinking they were both identical, but just not I tried pulling both the stable and the latest and it seems those are 2 different images (although maybe inside they are the same?). I don't know if this was the source of my issue.

Thanks
